### PR TITLE
Collect robot cfg instances

### PR DESCRIPTION
**Pull request by jontje**
As per title

### CODE REVIEW

File: include/abb_librws/rws_interface.h [hunk 1]
```diff
@@ -419,6 +419,13 @@ class RWSInterface
    */
   StaticInfo collectStaticInfo();
 
+  /**
+   * \brief Retrieves the configuration instances for the present robots.
```
Comment (gavanderhoorn) [inline include/abb_librws/rws_interface.h:423]: What do you mean by *present*? Would that be the robots connected to/configured on the controller the current RWS session communicates with?

If so, that might be a better description of what this method does.

Comment (jontje) [reply include/abb_librws/rws_interface.h]: Opps, `present robots` should be `robots defined in the system`.

Fixed in 42587d6

**Comment by jontje** [comment]
Synced with the base branch, and added error handling.

**Review by gavanderhoorn — APPROVED** [review]
Code is almost identical to that in #116?

**Comment by jontje** [comment]
> Code is almost identical to that in #116?

Yes, but I have only added extraction of data that I make use of (in downstream packages). There are many more attributes that could be extracted, especially for the `robot` configuration type, but those can be added on a need-to-use basis.

Also, some attributes (e.g. base frame data) could benefit of a common function to reduce the amount of code and the risk of mistakes.

**Comment by jontje** [comment]
Thanks for the reveiw @gavanderhoorn 👍

**Pull request merged**